test/src/org/apache/jmeter/protocol/http/util/TestHTTPArgument.java

```diff
@@ -45,7 +45,7 @@ public void testCloning() throws Exception {
             assertEquals("name.%3F", clone.getEncodedName());
             assertEquals("value_+here", clone.getEncodedValue());
             assertEquals("name.?", clone.getName());
-            assertEquals("value_+here", clone.getValue());
+            assertEquals("value_ here", clone.getValue());
```

Comment:
Why is this change? Is it intentional?

Reply:
This change was done in order to pass the test case testCloning. It wasn't supposed to be changed, was changed by mistake by us.